### PR TITLE
fix trt stream bug.

### PR DESCRIPTION
**Pull request by NHZlX**
BUG: After continuing to input different data, the output cannot be aligned

### CODE REVIEW

File: paddle/fluid/operators/tensorrt/tensorrt_engine_op_test.cc [hunk 1]
```diff
@@ -99,7 +99,7 @@ TEST(TensorRTEngineOp, manual) {
   SetAttr<std::string>(engine_op_desc.Proto(), "subgraph",
                        block_->SerializeAsString());
   SetAttr<int>(engine_op_desc.Proto(), "max_batch_size", 2);
-  SetAttr<int>(engine_op_desc.Proto(), "workspace_size", 2 << 10);
+  SetAttr<int>(engine_op_desc.Proto(), "workspace_size", 2 << 20);
```
Comment (Superjomn) [inline paddle/fluid/operators/tensorrt/tensorrt_engine_op_test.cc]: Why change to `2<<20` ?

**Review by Superjomn — COMMENTED** [review]
Add a test to check that different inputs and get right outputs.

File: paddle/fluid/operators/tensorrt/tensorrt_engine_op_test.cc (hunk 1 above)
Comment (NHZlX) [reply paddle/fluid/operators/tensorrt/tensorrt_engine_op_test.cc]: We should change it to 1 << 20 (the Official default). 1 << 10 is too small for trt5.

**Review by Superjomn — DISMISSED** [review]
LGTM

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**